Makefile

```diff
@@ -18,7 +18,9 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
-DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.14
+# release images are available at:
+# console.google.com/gcr/images/cloud-foundation-cicd/global/cft/developer-tools?project=cloud-foundation-cicd
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.15.9
```

Comment:
It used to be that this version of the docker image used for testing matches the terraform version. But lately this seems to have changed in the `cloud-foundation-toolkit`. So I had to keep bumping I found the image with terraform `0.15.5` was installed

Reply:
@Shabirmean for 0.13+ cft-devtools images, minor versions corresponds to the TF minor versions. Patch versions maybe updated for gcloud updates hence they maybe out of sync like https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/949. 

We also tag the image with TF minor version so, 0.15 would be 0.15.x where x is latest. For testing we tend to test at latest of a given minor TF version using this tag.

Reply:
Awesome. Thank you very much for clarifying this. ⚡